index.html

```diff
@@ -256,53 +256,112 @@ <h1>Terminology</h1>
+            <figcaption>Discovery architecture overview.  
+              A set of open Introduction mechanisms provides a set of URLs, which
+              point at Exploration services that only provide metadata (TDs) after suitable
+              authentication.  Thing Links and Thing Description Directories provide
+              additional flexibility but retrieving further results from these is
+              at the discretion of the application.
+            </figcaption>
         </figure>
-	<p>
-	The WoT discovery process is designed using a two-phase approach.
-	In the first phase, one or more of a set of "Introduction" mechanisms may be used to 
-	generate a set of candidate URLs.  These URLs do not themselves contain metadata,
-	but are used in the second stage, "Exploration", to actually fetch metadata.
-	The intention is that Introductions are relatively open and can support any
-	mechanism that can return a URL, including simply entering URLs directly.
-	Introductions, however, do not include any security or privacy controls and
-	so should not provide metadata directly.  Instead, the provided URLs are followed
-	to "Exploration" mechanisms that will actually provide metadata but allow for
-	suitable security and privacy controls.
+        <p>
+	       The intention is that Introductions are relatively open "first contact"
+         mechanisms to provide a starting point for the rest of the Discovery process.
+         In this document we specify details on several Introduction
+         mechanisms, suitable for different use cases, including both local and non-local
+         scenarios, but Introductions can in fact be provided any mechanism that can return a URL.
+	       Introductions, however, do not include any security or privacy controls and
+	       so should not provide metadata directly. 
+         Instead, the URLs provided by Introductions reference
+	       "Exploration" services.  Exploration services actually do provide metadata,
+         but only after suitable authentication and access controls have been applied.
+        </p>
+	      <p>
+	       The Discovery process can produce a <em>set</em> of URLs as output from
+	       its Introduction phase, even if only one Introduction mechanism is used
+         (some Introduction mechanisms can themselves return multiple URLs).
+	       The final output after the Exploration phase
+	       can also be a <em>set</em> of Thing Descriptions.
+        </p>
+        <p>
+	       Each URL provided by the Introduction phase always points at an 
+         Exploration service endpoint that will return a single Thing Description.
+	       In the simplest case this is the Thing Description of a Thing 
+	       describing an IoT endpoint device.  
+         In the case of self-describing Things an Introduction URL might
+	       even point directly at an endpoint provided by a 
+         Thing serving its own Thing Description.
```

Comment:
Two sentences telling the same thing?
```suggestion
         In the case of self-describing Things an Introduction URL might
	       even point directly at an endpoint provided by a 
         Thing serving its own Thing Description.
```